jmxtrans-output/jmxtrans-output-elastic/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jmxtrans</groupId>
+		<artifactId>jmxtrans-output</artifactId>
+		<version>251-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>jmxtrans-output-elastic</artifactId>
+
+	<name>JmxTrans - ElasticSearch output writers</name>
+
+	<properties>
+		<verify.mutationThreshold>80</verify.mutationThreshold>
```

Comment:
Let me think about the current fields in elastic. Maybe both host and port and server aliases can be added. As well as some other fields that are missing out now, such as classname.